1.3.c

```diff
@@ -1,121 +1,111 @@
 #include <stdio.h>
+#include <string.h>
 #include <stdbool.h>
+// I am did't do maxsize , because my IDE cant scanf this.
 
-#define MAXIMUMSIZE 256
-
-bool isCorrectString(char* string)
+int IncomingDataStatus(char symbol, char state) 

```

Comment:
```suggestion
int IncomingDataStatus(char symbol, int state) 
```
Вроде, состояние числом было, а не символом

Reply:
```suggestion
int updateStateBySymbol(char symbol, char state) 
```
функции с маленькой буквы, имя функции должно отражать то, что функция делает.
(-0.25)